app/views/annotation_categories/_annotation_category_edit.html.erb

```diff
@@ -20,6 +20,12 @@
                     AnnotationCategory.human_attribute_name(:annotation_category_name) %>
         <%= f.text_field :annotation_category_name, rows: '3' %>
       </div>
+      <br>
+      <div class='inline-labels'>
+        <%= f.label :flexible_criterion_id,
+                    t('annotation_categories.flexible_criterion_choice') %>
+        <%= f.collection_select :flexible_criterion_id, FlexibleCriterion.all, :id, :name, { include_blank: true } %>
```

Comment:
You should only be passing in the flexible criteria associated with the current assignment.

Reply:
So I fixed this (and for some reason it auto displays the current flexible_criterion when the page loads), however I wonder if this should be another aspect of validation in the annotation_category?